src/manifest/manifest_test.go

```diff
@@ -143,8 +150,10 @@ func TestManifestValidateNestedField(t *testing.T) {
 	m.Content.Branches = append(m.Content.Branches, &model.BranchManifest{
 		BranchKey: model.BranchKey{Id: 0},
 		Paths: model.Paths{
-			Path:       "foo",
-			ParentPath: "bar",
+			PathInProject: model.PathInProject{
```

Comment:
Z `model.Paths` som vyclenil strukturu `PathInProjet`, ktora obsahuje `Object a Parent path`.
Potreboval som to takto, kezde `Parent + Object path = relativna cesta k objektu v projekte` ... a ta musi byt unikatna.